### PR TITLE
Adding missing title to readme.md

### PR DESCRIPTION
**Pull request by kshithijiyer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# fedora Coreos Pipeline
```
Comment (bgilbert) [inline README.md]: If you fix the capitalization (`Fedora CoreOS Pipeline`) I'll merge it.  :slightly_smiling_face:

Comment (kshithijiyer) [reply README.md]: Done

**Comment by bgilbert** [comment]
Thanks!

**Comment by dustymabe** [comment]
Thanks @kshithijiyer

**Pull request merged**